### ISSUE TITLE
Template tab

### ISSUE THREAD

**Issue by Aryllen** [Sage-Bionetworks/dccvalidator#367]
Create a tab just for getting the templates. The person could select which one(s) are needed and be given links to the templates, instead of a single link to a folder of templates. We could potentially make this also work with HTAN's Google sheets links, in case users wanted that. However, we do need to be careful about people entering in PHI/PII in Google sheets.

Based on feedback from Duminda.

**Comment by karawoo** [comment]
We might consider combining this with #311 and giving people examples of filled-out files in addition to the templates.

**Comment by Aryllen** [comment]
I like that idea.

**Comment by Aryllen** [comment]
@kelshmo, @avanlinden, @RYaxley, @pitviper6, @amapeters...

We could add a template tab to dccvalidator for users to get their templates from. For templates that have registered JSON schema metadata templates, I could adapt the code Cindy wrote that creates the templates from those. The alternative is having a cron job that updates the templates in the Synapse folder. I believe Cindy has implemented this for PEC, but AD does not have this, yet.

There are pros and cons to both approaches. Before I move forward on adding this feature, do you all still want a dccvalidator tab for retrieving templates?

**Comment by avanlinden** [comment]
@Aryllen I like the idea of dccvalidator being a one stop shop for metadata templates and validation, so having a template tab in the app makes sense to me. It would also be great to encourage contributors who have uploaded data before to re-download the template each time they have a new study, so that they don't use out of date templates that trigger validation errors.

**Comment by kelshmo** [comment]
I love the idea of templates being available from dccvalidator and I especially love the idea of templates being generated in real time - to grab the most recent updates!

**Comment by pitviper6** [comment]
Yes to both of these things! The more we can consolidate the things data
contributors need in one place, and have them up to date and timely, the
better!

Juliane.
